www/lobby/id.ts

```diff
@@ -75,10 +98,13 @@ lobby_id_router.delete('/:id/user/:deleteId', async (req: Request, res: Response
   res.status(200).end();
 });
 
-const verifyIds = async (userId : string, lobbyId : string) : Promise<[User, Lobby] | null> => {
+const verifyIds = async (userId : string, lobbyId : string) : Promise<{user: User, lobby: Lobby} | null> => {
   const user = await User.fromId(userId);
   if (!user) return null;
   const lobby = await Lobby.fromId(lobbyId);
   if (!lobby) return null;
-  return [user, lobby];
+  return {
+    user: user,
+    lobby: lobby,
+  };
```

Comment:
nit: in javascript, the variable name will resolve as the key value of an object if you do it like this!

```suggestion
  return {
    user,
    lobby,
  };
```